lib/clever-ruby/district_admin.rb

```diff
@@ -0,0 +1,17 @@
+module Clever
+  # District Admin resource
+  class DistrictAdmin < APIResource
+    include Clever::APIOperations::List
+    @uri = 'district_admins'
+    @plural = 'district_admins'
+    @linked_resources = []
+
+    # Optional attributes
+    # @see Clever::CleverObject.optional_attributes
+    # @api private
+    # @return [Array]
+    def optional_attributes
+      [:name, :title]
```

Comment:
I was looking at other files in the same directory to see what they had for this and it appears that it thinks School Admins need a name but don't need an email, but that doesn't jive with what the clever data schema table said. I wonder what's up with that?